docs/README.md

```diff
@@ -5,5 +5,4 @@ Run `scripts/document.sh`
 ## Update the documentation site:
 1. Clone [mapbox-navigation-ios](https://github.com/mapbox/mapbox-navigation-ios) to a mapbox-navigation-ios-docs folder alongside your MapboxDirections.swift clone, and check out the `mb-pages` branch.
 1. In your main MapboxDirections.swift clone, check out the release branch and run `OUTPUT=../mapbox-navigation-ios-docs/directions/X.X.X scripts/document.sh`, where _X.X.X_ is the new SDK version.
```

Comment:
Maybe we should refactor the script so that `OUTPUT` is the path to directions/ and the version subfolder is determined automatically.